tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor default values.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+#define SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+
+#include "accessor_common.h"
+#include "catch2/catch_test_macros.hpp"
```

Comment:
We are currently using Catch2 as a major architecture dependency, so not sure if we actually need this one

Reply:
this include has been removed [here](https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/c040cc7560db878fb467d794a504c6e5e20a0d0e#diff-9ed3eae8f2889bc04dfc0fad247a8fa4e381e1fff4144e0f1a23670d108f6fe9L12)